src/hub.js

```diff
@@ -206,7 +206,7 @@ window.APP.RENDER_ORDER = {
 };
 
 const store = window.APP.store;
-store.update({ preferences: { shouldPromptForRefresh: undefined } }); // Clear flag that prompts for refresh from preference screen
+store.update({ preferences: { shouldPromptForRefresh: false } }); // Clear flag that prompts for refresh from preference screen
```

Comment:
Preference default for `shouldPromptForRefresh` is `false`, so setting this to `false` here will result in `shouldPromptForRefresh` being stripped from preferences.